src/app/pages/auth/login/otpVerify.tsx

```diff
@@ -0,0 +1,124 @@
+"use client"
```

Comment:
If the content of these codes are just a compent for a certain page, move them under components directory. And also don't use pages anywhere else. You can move this file into `scr/components/VerifyOTP` for example.